setup.py

```diff
@@ -126,9 +126,8 @@ def find_version(*file_paths):
 
 
 TF_EXTRAS = [
-    "tensorflow~=2.11.1",
-    # The workaround of the protobuf issue and should be fixed with migration on TF 2.12
-    "tensorflow-metadata<=1.13.0",
+    "tensorflow~=2.12.0",
+    "protobuf<=3.20.3"
```

Comment:
The workaround is not fixed by upgrading to TF 2.12. I've added an explicit requirement `protobuf<=3.20.3`, because this package being upgraded to `4.*` actually causes problems. This way we can both make it work and use the latest version of `tensorflow-metadata` which is `1.13.1`.